pr_agent/settings/pr_code_suggestions_prompts.toml

```diff
@@ -79,17 +81,21 @@ code_suggestions:
 {%- if summarize_mode %}
   existing_code: |-
     def func1():
+  {%- if include_improved_code %}
   improved_code: |-
     ...
+  {%- endif %}
   one_sentence_summary: |-
     ...
   relevant_lines_start: 12
   relevant_lines_end: 12
 {%- else %}
   existing_code: |-
     def func1():
+{%- endif %}
   relevant_lines_start: 12
   relevant_lines_end: 12
+{%- if include_improved_code %}
   improved_code: |-
     ...
```

Comment:
this section is implemented wrongly

Reply:
Could you elaborate, please? It looks correct to me as I am including the pattern for "improved_code" only when the option is set.